Sources/DbSearch-postgresql.php

```diff
@@ -40,7 +40,7 @@ function db_search_init()
  */
 function smf_db_search_support($search_type)
 {
-	$supported_types = array('custom');
+        $supported_types = array('fulltext');
```

Comment:
You don't want to remove 'custom' from that array. Doing so will prevent the "custom" search index from working on new installs and will cause all kinds of problems for anyone upgrading.